snap/snapcraft.yaml

```diff
@@ -60,18 +59,11 @@ apps:
       - avahi-observe
       - browser-support
       - cups-control
-      - gsettings
       - opengl
-      - home
       - desktop
       - desktop-legacy
-      - mount-observe
       - network
       - network-bind
-      - network-observe
```

Comment:
I think Chromium-based snaps need this one for `navigator.onLine` but we aren't using that anymore because it was returning TRUE as long as you were connected to a network of some kind (even if the network had no internet). Seems safe to remove!